firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/FirebaseAppDistributionTest.java

```diff
@@ -475,4 +475,20 @@ public void updateApp_withApkReleaseAvailable_returnsSameApkTask() {
 
     assertEquals(updateTask, updateTaskToReturn);
   }
+
+  private UpdateTask updateToNewRelease(AppDistributionReleaseInternal newRelease) {
```

Comment:
I like the consolidation of setup, but this is combining some of the setup with the actual actions being tested. Could we split out the mocking/caching from the `updateIfNewReleaseAvailable()` call?

Reply:
Good call. Done. After removing the setCachedNewRelease call below, I felt like this wasn't doing enough to justify a separate method.

I moved the signin mocking to the setup method, since most tests aren't testing the signIn logic